testing/php7-memcached/APKBUILD

```diff
@@ -4,6 +4,7 @@ pkgname=php7-memcached
 _pkgreal=memcached
 _pkgrepo=php-memcached
 _pkgbranch=php7
+_pkgcommit=583ecd68faec886ac9233277531f78fb6e2043c7
 pkgver=3.0.0b1
```

Comment:
This is a snapshot, so the version should be something like `3.0_pre20160923`. And you should also try to request a release from the upstream, at least open an issue on their issue tracking…